docs/topics/secured-mission-intro.adoc

```diff
@@ -21,38 +25,38 @@ a|
 The default configuration includes both public and **private keys in the yaml configuration files**.
 
 a|
-We did this because the end user can deploy Red Hat SSO module and have it in a usable state without needing to know the internals or how to configure Red Hat SSO.
+Allows the end user to deploy the Red Hat SSO module without needing to know the internals or how to configure Red Hat SSO.
 
 a|
-In production, do not store private keys under source control. They should be added by the server administrator.
+In production, do not store private keys under version control. They should be added by the server administrator.
 
 
 a|
 The configured **clients accept any callback url**.
 
 a|
-To avoid having a custom configuration per runtime we avoid the verification of the callback as mandated by the OAuth2 specification.
+This example does not perform verification of the callback as mandated by the OAuth2 specification. A default configuration suitable for all runtimes was used.
 
 a|
-An application-specific callback URL should be provided with a valid domain name.
+In production, an application-specific (custom) callback URL with a valid domain name should be provided.
 
 a|
-**Clients do not require SSL/TLS and the secured applications are not exposed over HTTPS**.
+**Clients do not require SSL/TLS authentication and the secured applications are not exposed over HTTPS**.
 
 a|
-The boosters are simplified by not requiring certificates generated for each runtime.
+The boosters are simplified and do not require certificates to be generated for each runtime.
 
 a|
-In production a secure application should use HTTPS rather than plain HTTP.
+In production, a secure application should use HTTPS rather than standard HTTP.
 
 a|
-**The token timeout has been increased to 10 minutes from the default of 1 minute.**
+**The token timeout has been increased from 1 minute to 10 minutes.**
```

Comment:
Lost the information that 1 minute is the default. Without the word "default", I'm just asking myself "OK, so where does the 1 minute come from?".